source/platforms/github/GitHubAPI.ts

```diff
@@ -96,7 +95,7 @@ export class GitHubAPI {
       .map(comment => comment.id) // only return IDs
   }
 
-  updateCommentWithID = async (id: number, comment: string): Promise<any> => {
+  updateCommentWithID = async (id: string, comment: string): Promise<any> => {
```

Comment:
The existing `Comment` type has its ids as strings, and I _think_ that these are only internal API, so I figured that this change was okay. Whaddayathink?

Reply:
That makes sense 👍